Dockerfile.base

```diff
@@ -15,7 +15,7 @@
 FROM ubuntu:jammy
 
 # Update and install libraries and tools from repositories
-RUN apt-get update && \
+RUN apt-get update && apt-get upgrade libpam0g=1.4.0-11ubuntu2.4 && \
```

Comment:
Would it be enough to retrigger a build of the base image (since presumably `libpam` would get updated to 1.4.0 _or newer_ anyway by that)? Or only doing `apt-get upgrade`? I'd like to avoid `libpam` getting stuck on 1.4.0 when there will be newer versions available.